pkgs/development/tools/misc/watson-ruby/default.nix

```diff
@@ -0,0 +1,18 @@
+{ lib, bundlerEnv, ruby }:
+
+bundlerEnv rec {
+  name = "watson-ruby-${version}";
+
+  version = (import ./gemset.nix).watson-ruby.version;
+  inherit ruby;
+  # expects Gemfile, Gemfile.lock and gemset.nix in the same directory
+  gemdir = ./.;
+
+  meta = with lib; {
+    description = "An inline issue manager";
+    homepage    = http://goosecode.com/watson/;
+    license     = with licenses; mit;
+    maintainers = with maintainers; [ robertodr ];
```

Comment:
You also need to declare this maintainer in `lib/maintainers.nix`.

Reply:
Maintainer declared. I also sorted the list of maintainers.